### PR TITLE
feat: add get avaiable elections endpoint

### PR DESCRIPTION
**Pull request by frattezi**
# Why is this PR necessary, what does it do?

This PR startup our nextjs API already communicating with Notion API. It is the first of a series of pull requests and adds the `getAvaiableElections` handler, which will return all valid elections from a personal notion database.

**Sample API return**
```json
{
    "next_cursor": null,
    "results": [
        {
            "electionId": "47b8e344-0243-435d-822b-4192f691f5a7",
            "electionName": "Deputado estadual - Sagrado Coração"
        },
        {
            "electionId": "58ebc9cd-5a4d-4902-876c-95008a537657",
            "electionName": "Colegio chacha 21/04"
        }
    ],
    "message": "2 pages found"
}
```

## Observations
The notion API SDK is really hard to operate, but after thinking about it, it would be harder to type and check everything without it. Due to this, I had to create a `/utils/notion.ts` helper and `entities/notion-sdk` which should never be touched, only for a future removal when some notion-sdk issues are resolved.

Also, some features are needed, add an error handler, map our API interfaces using open API etc...
Due to our time limitation, I'm only delivering the best-case scenario for the API, already modeling the needed data by the client. I will also add some new issues regarding those topics

Relates to #2

### CODE REVIEW

File: pages/api/elections/index.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { getAvaiableElections } from '@packages/notion-client';
+
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<AvaiableElectionsReturn>,
+) {
+  const { databaseId } = req.query;
+  if (!(databaseId instanceof Array)) {
```
Comment (marco-souza) [inline pages/api/elections/index.ts:11]:
```suggestion
  if (!Array.isArray(databaseId)) {
```

File: packages/entities/notion-sdk.ts [hunk 2]
```diff
@@ -0,0 +1,4659 @@
+// TODO(Frattezi): Since the notion-sdk don't export its types we need
```
Comment (marco-souza) [inline packages/entities/notion-sdk.ts:1]: Acho que esse cara deveria estar em `packages/notion/sdk.ts`, pois não é uma entidade do sistema.

File: packages/notion-client.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { Client } from '@notionhq/client';
```
Comment (marco-souza) [inline packages/notion-client.ts:1]: Also should be moved to `packages/notion/client.ts`. Só pra centralizar mais as coisas relacionadas ao notion.

File: packages/utils/notion.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import { Title, LooseObject, Result } from '@packages/entities/notion-sdk';
```
Comment (marco-souza) [inline packages/utils/notion.ts:1]: Same `packages/notion/utils.ts`

File: packages/notion-client.ts (hunk 3 above)
Comment (frattezi) [reply packages/notion-client.ts]: done

File: packages/utils/notion.ts (hunk 4 above)
Comment (frattezi) [reply packages/utils/notion.ts]: done

**Review by marco-souza — APPROVED** [review]
lgtm!

**Pull request merged**